index.md

```diff
@@ -8,6 +8,469 @@ This is some welcome text.
 
 This is some helpful description of the C0111 error.
 
+### C0102 Blacklisted name {#C0102}
+
+This error occurs when a variable name is listed in the black list. The black 
+list includes names like foo, bar, baz, toto, tata, tutu and so on. Students 
+are expected to use good variable names such as i, j, k, ex, Run. The example 
+below uses blacklisted names.
+
+```python 
+def pos(obj):
+    """
+    @type obj: int
+    @rtype: bool
+    """
+    foo = obj
+    if foo < 0:
+        return False
+    else:
+        return True
+```
+
+### C0103 Invalid Name {#C0103}
+
+This error occurs when an invalid name is used as a variable name. This type of 
+error is a sign that the variable names should be modified to be of the form: a 
+lowercase letter followed by a lowercase letter or digit. The minimum number of 
+characters should be 2 and the maximum should be 30.
+
+```python
+def is_equal(obj):
+    """
+    @type obj: int
+    @rtype: bool
+    """
+    Ex = 5
+    return obj == Ex
+```
+
+### C0111 Missing Docstring {#C0111}
+
+This error occurs when a module, function, class or method has no docstring. 
+Some special methods like __init__ don't require a docstring. Students are 
+expected to write docstrings for every module, function, class and method (maybe
+ except the special methods with '__').
+
+```python
+def is_true(obj):
+    if obj is False:
+        return False
+    else:
+        return True
+```
+
+### C0112 Empty Docstring {#C0112}
+
+This error occurs when a module, function, class or method has an empty 
+docstring. Students are expected to write complete docstrings for every module, 
+function, class and method.
+
+```python
+def is_true(obj):
+    """
+    """
+    if obj is False:
+        return False
+    else:
+        return True
+```
+
+### C0113 Unneeded not {#C0113}
+
+### C0121 Singleton comparison {#C0121}
+
+This is an error that occurs when an expression is compared to singleton values 
+like True, False or None. This type of error indicates incorrect comparison 
+tests, so please check your code carefully.
+
+```python
```

Comment:
This example should be improved (see my comment in your other pull request), but also please include here a *corrected* version of the example as well.